Makefile

```diff
@@ -3,28 +3,41 @@
 # Please refer to the LICENSE and AUTHORS files for details
 # SPDX-License-Identifier: MIT 
 
-CC := gcc
+CC     := gcc
+CFLAGS := -DPT_DEBUG -O2 -std=c99 -pedantic -Wall -Wextra
+
+LUA_DIR := /usr
+
 INSTALL_DIR    := /usr/local
 INSTALL_INCDIR := $(INSTALL_DIR)/include
 INSTALL_LIBDIR := $(INSTALL_DIR)/lib
+INSTALL_BINDIR := $(INSTALL_DIR)/bin
 
-.PHONY: install ptracer_header libptracer uninstall clean
+.PHONY: install ptracer_header pt-run libptracer uninstall clean
 
-install: ptracer_header libptracer
+install: ptracer_header pt-run libptracer
+	strip -s pt-run
+	cp pt-run $(INSTALL_BINDIR)
+	strip -s libptracer.so
 	cp libptracer.so $(INSTALL_LIBDIR)
 
 # We need the `ptracer.h` header to be installed first.
 ptracer_header: 
 	cp lib/ptracer.h $(INSTALL_INCDIR)
 
+pt-run: 
+	$(CC) $(CFLAGS) src/pt-run/main.c -o pt-run -llua -lm -Wl,-E -L$(LUA_DIR)/lib
+
 libptracer:
-	$(CC) -fPIC -DPT_DEBUG -O2 -shared src/ptracer/main.c -o libptracer.so
+	$(CC) -fPIC -shared $(CFLAGS) src/ptracer/main.c -o libptracer.so
```

Comment:
I thought we were getting rid of libptracer.so? I'm not looking forward to maintaining a shared library that is hard to install and that we do not use...

Reply:
Now that I think about it, our actual source code is really small rn. Static linking should be fine. Let's do Makefile improvements and other nit-fixing in next PR.